### DISCUSSION TITLE
Bug in Announcements category ?

### DISCUSSION THREAD

**Discussion by prudhomm** [community/community#4063] [Discussions] [answered]
It seems I cannot create discussion in Announcements category type even though I am admin and maintainer.

We can add a new category Announcement 
![Screenshot 2021-05-30 at 11 32 21](https://user-images.githubusercontent.com/151693/120099221-b55d8100-c13a-11eb-80c9-51a499011ada.png)

but when starting an Announcement discussion is not possible, the Announcements category does not appear

![Screenshot 2021-05-30 at 11 32 38](https://user-images.githubusercontent.com/151693/120099230-c0181600-c13a-11eb-89f5-c9fe1cdf0325.png)

Am I missing something ?

I am the maintainer and admin of the repo

**Comment by pattacini** [comment]
I can confirm that I'm experiencing the same behavior.

**Reply by abrokenjester** [reply]
As am I, though on a repo where I am maintainer, but not admin.

**Comment by abrokenjester** [comment]
This isn't a complete answer, but I've managed to find a workaround. Even though I cannot create a new discussion _directly_ in the Announcements category, it turns out to be possible to create the new discussion in some other category, and then reassign the category to move it to Announcements.

**Comment by laymonage** [comment, marked as answer]
According to https://github.com/github/feedback/discussions/2937#discussioncomment-818904, this should be fixed by now.

**Reply by pattacini** [reply]
Yes! Working now!

**Reply by sarmadgardezi** [reply]
I have the same issue but it crash sometime.
